subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -1153,6 +1191,15 @@ uint8_t ull_cp_remote_dle_pending(struct ll_conn *conn)
 }
 #endif /* CONFIG_BT_CTLR_DATA_LENGTH */
 
+uint8_t ull_cp_remote_fsu_pending(struct ll_conn *conn)
+{
+	struct proc_ctx *ctx;
+
+	ctx = llcp_rr_peek(conn);
+
+	return (ctx && ctx->proc == PROC_FRAME_SPACE);
```

Comment:
Mismatch between types of the function (`uint8_t) and the return values (`bool`)